libraries/report.rb

```diff
@@ -28,13 +28,32 @@ class ComplianceReport < Chef::Resource
       blob[:reports] = reports
       blob[:profiles] = ownermap
 
-      Chef::Config[:verify_api_cert] = false
-      Chef::Config[:ssl_verify_mode] = :verify_none
+      # resolve owner
+      o = return_or_guess_owner
 
-      url = construct_url(::File.join('/organizations', org, 'inspec'))
-      # Chef::Log.debug "url: #{url}"
-      rest = Chef::ServerAPI.new(url, Chef::Config)
-      rest.post(url, blob)
+      if token
+        if direct
```

Comment:
instead of direct we may use `:chef_server` & `:chef_compliance` as targets with default to chef server. What do you think?